### PR TITLE
Add Amethyst Game Engine

### PR DESCRIPTION
**Pull request by khionu**

### CODE REVIEW

**Comment by SolarionFX** [comment]
I'd be happy to accept this PR, I just need you to resolve the conflicts!

**Comment by khionu** [comment]
There, fixed

File: communities.json [hunk 1]
```diff
@@ -562,6 +562,13 @@
       "quote": "Discord helps us provide immediate support to our players and keeps us in touch with them every day in a way that is far more personal and social than our forums and subreddit.",
       "inviteCode": "ECJk293",
       "githubUrl": "https://github.com/TES3MP/openmw-tes3mp"
+    },
+    {
+      "logo": "amethyst.png",
+      "title": "Amethyst Game Engine",
+      "quote": "Discord has made communicating with and helping our users in real time a breeze. Having an modern and easily accessible chat has been critical in our growth.",
```
Comment (Moxinilian) [inline communities.json]: Shouldn't it be "a modern" here?

**Comment by khionu** [comment]
My pleasure!

**Pull request merged**